### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by irfankhan10**
pre-release

### CODE REVIEW

File: src/custom/docs/components/navmenu/schema.tsx [hunk 1]
```diff
@@ -1,18 +1,25 @@
...
+    href: z.string(),
+    title: z.string(), 
+});
 const linkSchema = z.object({
     href: z.string(),
     title: z.string(),
+    logo: z.custom<React.JSX.Element>(
+        e => (e as any)?.$$typeof === Symbol.for("react.element"),
+        "value must be a React Element"
+    ),
```
Comment (aidanCQ) [inline src/custom/docs/components/navmenu/schema.tsx:15]:
```suggestion
    logo: z.custom<React.JSX.Element>(
        e => (e as any)?.$$typeof === Symbol.for("react.element"),
        "value must be a React Element"
    ),
```
🧐

File: src/custom/docs/components/navmenu/NavigationMenu.tsx [hunk 2]
```diff
@@ -23,15 +25,37 @@ export const Navigation = (props: {
...
+                    <NavigationMenuLink asChild>
+                      <RegularLink
+                        className="flex h-full w-full select-none flex-col justify-end rounded-md bg-gradient-to-b from-muted/50 to-muted p-6 no-underline outline-none focus:shadow-md"
+                        href={item.href}
+                      >
+                        {item.logo}
+                        <p className="text-sm leading-tight text-muted-foreground py-6">
+                          {item.description}
+                        </p>
+                      </RegularLink>
```
Comment (aidanCQ) [inline src/custom/docs/components/navmenu/NavigationMenu.tsx:41]:
```suggestion
                      <RegularLink
                        className="flex h-full w-full select-none flex-col justify-end rounded-md bg-gradient-to-b from-muted/50 to-muted p-6 no-underline outline-none focus:shadow-md"
                        href={item.href}
                      >
                        {item.logo}
                        <p className="text-sm leading-tight text-muted-foreground py-6">
                          {item.description}
                        </p>
                      </RegularLink>
```
If regularLink is an a tag, I would just use an a tag instead

File: src/custom/docs/components/navmenu/index.tsx [hunk 3]
```diff
@@ -1,57 +1,164 @@
...
-            </div> : null}
+            <div className="text-muted-foreground text-xs font-medium flex items-center gap-1.5">
+              <div className='mx-0.5 text-muted-foreground/50'>|</div><div>Documentation</div>
+            </div>
           </div>
-          <Link href="/" className="ml-4 mr-4 flex items-center space-x-2">
+          <RegularLink href="/" className="ml-4 mr-4 flex items-center space-x-2">
             <span className="hidden font-bold">Quantinuum</span>
-          </Link>
+          </RegularLink>
```
Comment (aidanCQ) [inline src/custom/docs/components/navmenu/index.tsx:156]: a tag here

File: src/custom/docs/components/navmenu/schema.tsx (hunk 1 above)
Comment (irfankhan10) [reply src/custom/docs/components/navmenu/schema.tsx:15]: I have removed

File: src/custom/docs/components/navmenu/NavigationMenu.tsx (hunk 2 above)
Comment (irfankhan10) [reply src/custom/docs/components/navmenu/NavigationMenu.tsx:41]: done

File: src/custom/docs/components/navmenu/index.tsx (hunk 3 above)
Comment (irfankhan10) [reply src/custom/docs/components/navmenu/index.tsx:156]: removed

File: src/custom/docs/components/navmenu/NavigationMenu.tsx (hunk 2 above)
Comment (aidanCQ) [reply src/custom/docs/components/navmenu/NavigationMenu.tsx:41]: https://github.com/CQCL/quantinuum-ui/pull/34/commits/5ed49a3245968541b606884a33d1fa041e0b2a42

File: src/custom/docs/components/navmenu/index.tsx (hunk 3 above)
Comment (irfankhan10) [reply src/custom/docs/components/navmenu/index.tsx]: https://github.com/CQCL/quantinuum-ui/pull/34/commits/05de846bd0797896edb8ca5328f51abccebb4d9e

File: src/custom/docs/components/navmenu/NavigationMenu.tsx (hunk 2 above)
Comment (irfankhan10) [reply src/custom/docs/components/navmenu/NavigationMenu.tsx]: https://github.com/CQCL/quantinuum-ui/commit/05de846bd0797896edb8ca5328f51abccebb4d9e

File: src/custom/docs/components/navmenu/schema.tsx (hunk 1 above)
Comment (irfankhan10) [reply src/custom/docs/components/navmenu/schema.tsx]: https://github.com/CQCL/quantinuum-ui/pull/34/commits/97879675293df7994f881904539c04410e1076e6

**Review by aidanCQ — APPROVED** [review]

**Pull request merged**